### ISSUE TITLE
send confirmation e-mail to User after they submit a changeset and after changeset is applied

### ISSUE THREAD

**Issue by drewda** [transitland/transitland-datastore#281]
e-mail copy: #325 and #326

I'll take a quick look at https://mandrill.zendesk.com/hc/en-us/articles/205582507-Getting-Started-with-Templates and see if that would enable @rmglennon to edit copy... or if it's more trouble than it's worth and best to just use ActiveMailer templates in Rails

After #258

**Comment by drewda** [comment]
Drew, @mjcunningham, @baldur talk about using Mailchimp, Mandrill, or AWS SES.

**Comment by baldur** [comment]
@drewda I vote for using mailchimp or mandril as it keeps me out of it :+1:

**Comment by drewda** [comment]
@baldur sounds good. I'll follow up on Slack for creds.

**Comment by drewda** [comment]
Mandrill set-up: https://github.com/mapzen/operations-engineering/issues/58

**Comment by rmglennon** [comment]
looks like #326 may be a duplicate of this? See that issue for some text for the email message.
